packages/@okta/vuepress-site/docs/concepts/saml/index.md

```diff
@@ -13,7 +13,7 @@ However, with increased collaboration and the move towards the cloud, many appli
 
 ## Authentication
 
-Before diving into federated authentication, we need to understand what authentication really means. Authentication defines the way a user is identified and validated through some sort of credentials as part of a sign-in flow. Most applications present a sign-in page to an end user, allowing him to specify a username and a password. In some cases, additional information may be required to locate the user - like a company ID or a client code. This information allows the application to narrow down the search of the username applicable to the provided info. This is often used to allow the same username to exist across multiple tenants belonging to different customers.
+Before diving into federated authentication, we need to understand what authentication really means. Authentication defines the way a user is identified and validated through some sort of credentials as part of a sign-in flow. Most applications present a sign-in page to an end user, allowing the user to specify a username and a password. In some cases, additional information may be required to locate the user - like a company ID or a client code. This information allows the application to narrow down the search of the username applicable to the provided info. This is often used to allow the same username to exist across multiple tenants belonging to different customers.
```

Comment:
In some cases, additional information may be required to locate the user, like a company ID or a client code.

In our Global English guide - we have information about not using em dashes as a formatting device. I know that we've implemented these in some cases when we discuss parameters in a list, but here in text, we should avoid them. 

"There is nothing inherently wrong with using em dashes in these contexts. However, if we sometimes use em dashes and other times use parentheses to set off the same type of information, the inconsistency interferes with the efficient use of translation memory.”

So basically. It’s not wrong, but it’s also not common. Better to just use a comma and not risk introducing a weird formatting idiosyncracy.

Reply:
Sure thing - I can make this change.
(In my defense, I haven't rewritten this content at all. This was just to remove the use of "him" that someone found.)